index.d.ts

```diff
@@ -0,0 +1,138 @@
+declare module 'pmcrypto' {
```

Comment:
When authoring definitions for libraries, you shouldn't declare it by name, it's by default going to be resolved correctly automatically. Since pmcrypt doesn't declare a global, you can just remote the `declare module 'pmcrypto' {` part and it should work fine.

cf https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html